dotnet-9.yaml

```diff
@@ -1,7 +1,8 @@
+#nolint:git-checkout-must-use-github-updates
 package:
   name: dotnet-9
-  version: 9.0.101
-  epoch: 2
+  version: 9.0.200
```

Comment:
Whats the reason we can't use 9.0.2 here and start fresh with the correct version? it'll be newer than 9.0.101, and means we can continue to use GH to poll for updates?

Reply:
We can't use GitHub to pill for updates because 101 > 2, regardless of the age of the tag. Even if we withdrew 101, we'd still be stuck using release monitor